modules/advanced-node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,154 @@
+
+.Procedure
+
+. Create a config map that contains a tuned object manifest for creating 10 huge pages that are 2 MB in size. Save this config map manifest in a file named `tuned-hugepages.yaml`:
++
+[source,yaml]
+----
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-hugepages
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: hugepages
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Boot time configuration for hugepages
+              include=openshift-node
+              [bootloader]
+              cmdline_openshift_node_hugepages=hugepagesz=2M hugepages=50
+            name: openshift-node-hugepages
+          recommend:
+          - priority: 20
+            profile: openshift-node-hugepages
+----
++
+[NOTE]
+====
+The `.spec.recommend.match` field is intentionally left blank. In this case, this tuned is applied to all nodes in the node pool where this config map is referenced. Group nodes with the same hardware configuration into the same node pool. Otherwise, TuneD operands can calculate conflicting kernel parameters for two or more nodes that share the same node pool.
+====
+
+. Create the config map in the management cluster:
++
+[source,terminal]
+----
+$ oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-hugepages.yaml
+----
+
+. Create a node pool manifest YAML file, customize the upgrade type of the node pool, and reference the config map that you created in the `spec.tunedConfig` section. Create the node pool manifest and save it in a file named `hugepages-nodepool.yaml`:
++
+[source,yaml]
+----
+    NODEPOOL_NAME=hugepages-example
+    INSTANCE_TYPE=m5.2xlarge
+    NODEPOOL_REPLICAS=2
+
+    hypershift create nodepool aws \
+      --cluster-name $CLUSTER_NAME \
+      --name $NODEPOOL_NAME \
+      --node-count $NODEPOOL_REPLICAS \
+      --instance-type $INSTANCE_TYPE \
+      --render > hugepages-nodepool.yaml
+----
+
+. In the `hugepages-nodepool.yaml` file, set `.spec.management.upgradeType` to `InPlace`, and set `.spec.tunedConfig` to reference the `tuned-hugepages` config map that you created.
++
+[source,yaml]
+----
+    apiVersion: hypershift.openshift.io/v1alpha1
+    kind: NodePool
+    metadata:
+      name: hugepages-nodepool
+      namespace: clusters
+      ...
+    spec:
+      management:
+        ...
+        upgradeType: InPlace
+      ...
+      tunedConfig:
+      - name: tuned-hugepages
+----
++
+[NOTE]
+====
+To avoid the unnecessary re-creation of nodes when you apply the new 'MachineConfig' objects, set `.spec.management.upgradeType` to `InPlace`. If you use the `Replace` upgrade type, nodes are fully deleted and new nodes can replace them when you apply the new kernel boot parameters that the TunedD operand calculated.
```

Comment:
```suggestion
To avoid the unnecessary re-creation of nodes when you apply the new 'MachineConfig' objects, set `.spec.management.upgradeType` to `InPlace`. If you use the `Replace` upgrade type, nodes are fully deleted and new nodes can replace them when you apply the new kernel boot parameters that the TuneD operand calculated.
```

Reply:
@lahinson I think this typo is still in the docs.